VSRAD.Package/DebugVisualizer/Breakpoint.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+
+namespace VSRAD.Package.DebugVisualizer
+{
+    public struct Breakpoint : System.IEquatable<Breakpoint>
```

Comment:
I think it's better to make this a `class` so that the `Resumable` property could be mutated, that would simplify a lot of code down the line.

Reply:
Yes, it really is better